pytype/tools/analyze_project/config.py

```diff
@@ -186,8 +193,11 @@ def generate_sample_config_or_die(filename, pytype_single_args):
   items = dict(ITEMS)
   assert set(_PYTYPE_SINGLE_ITEMS) == set(pytype_single_args)
   for key, item in _PYTYPE_SINGLE_ITEMS.items():
-    items[key] = item._replace(default=pytype_single_args[key].default,
-                               comment=pytype_single_args[key].help)
+    if items[key].comment is None:
```

Comment:
This should be `item.comment`, not `items[key].comment`.

Reply:
Done. Code simplified.